src/LinuxTracing/SwitchesStatesNamesVisitorTest.cpp

```diff
@@ -201,7 +255,17 @@ ::testing::Matcher<ThreadStateSlice> ThreadStateSliceEq(const ThreadStateSlice&
       ::testing::Property("thread_state", &ThreadStateSlice::thread_state, expected.thread_state()),
       ::testing::Property("duration_ns", &ThreadStateSlice::duration_ns, expected.duration_ns()),
       ::testing::Property("end_timestamp_ns", &ThreadStateSlice::end_timestamp_ns,
-                          expected.end_timestamp_ns()));
+                          expected.end_timestamp_ns()),
+      ::testing::Property("wakeup_reason", &ThreadStateSlice::wakeup_reason,
+                          expected.wakeup_reason()),
+      ::testing::Property("wakeup_tid", &ThreadStateSlice::wakeup_tid, expected.wakeup_tid()),
+      ::testing::Property("wakeup_pid", &ThreadStateSlice::wakeup_pid, expected.wakeup_pid()),
+      ::testing::Property("switch_out_or_wakeup_callstack_id",
+                          &ThreadStateSlice::switch_out_or_wakeup_callstack_id,
+                          expected.switch_out_or_wakeup_callstack_id()),
+      ::testing::Property("switch_out_or_wakeup_callstack_status",
```

Comment:
This should be moved up as I think we moved this field up in the proto itself.

Reply:
Done.